tests/unit/suites/libraries/cms/router/JRouterTest.php

```diff
@@ -132,7 +132,8 @@ public function testLegacyApplicationRouterIsStillLoaded()
 			'id'   => 3,
 			'name' => 'tester',
 			'path' => __DIR__ . '/data'
-		));
+			)
```

Comment:
So technically this is "Closing parenthesis of a multi-line function call must be on a line by itself which is the PEAR.Functions.FunctionCallSignature.CloseBracketLine rule. I somewhat agree that it looks weird but that is the rule our coding standard applies. It's the same rule in phpcs 2.x as the old custom sniff [Joomla.Functions.FunctionCallSignature.CloseBracketLine](https://github.com/joomla/coding-standards/blob/master/Sniffs/Functions/FunctionCallSignatureSniff.php#L249)